### ISSUE TITLE
The detach message should include the RWX volume behavior

### ISSUE THREAD

**Issue by khushboo-rancher** [longhorn/longhorn#2059]
**Describe the bug**
The RWX volume can't be detach using Longhorn UI. On clicking detach, the volume first get detached and then automatically get attached to the volume.

**To Reproduce**
1. Go to Longhorn UI.
2. Select a RWX volume and detach it.
3. The volume get detached and attached again.
4. The message doesn't reflect this behavior.

**Expected behavior**
The message should display this behavior.

**Environment:**
 - Longhorn version: Longhorn v1.1.0-rc1

**Comment by yasker** [comment]
@joshimoo Can you clarify what's expected in this situation? So we can update the UI with the message.

**Comment by joshimoo** [comment]
If a workload is using a shared volume (app-x) then the share-manager for that volume will be in control of the volume.
So when the user force detaches the volume, the share-manager pod will detect this and terminate itself which will trigger an error
on the share-manager controller which will then ramp up a new pod and ensure that the volume is attached.

tldr something like, `detaching a shared volume will lead to automatic reattachment as long as there are workloads depending on that volume, ensure that the workloads are scaled down before trying to detach the volume`

**Comment by smallteeths** [comment]
I think it would be better to return an error message after the deatch request. @yasker @joshimoo

**Comment by joshimoo** [comment]
@smallteeths if I understand the pr correctly you are disabling the detach button and adding a tooltip that will be showed while browsing to the button? I think that's a good approach since it doesn't make sense to detach an active in use volume :)

**Comment by khushboo-rancher** [comment]
With Longhorn-ui `image: wangsiye/longhorn-ui:fdfce44`, the button is disabled for `rwx` volume and the tooltip is available.

Looks good to me.

<img width="1762" alt="Screen Shot 2021-01-08 at 12 13 55 PM" src="https://user-images.githubusercontent.com/60111667/104060019-219b1080-51ab-11eb-9b53-2ab93733e8e4.png">

**Comment by khushboo-rancher** [comment]
Validate with Longhorn Master - `01/11/2021`

Validation - **Pass**

The detach button is disabled for `rwx` volume and there is a tool tip to show the error message.
